formulas/github-get-archive/README.md

```diff
@@ -0,0 +1,24 @@
+github-get-archive
+==================
+
+This formula adds a function that allows you to pull down a github repository in tar format.
+
+    <repository>        Path to Github repository.
+
+    --tag=TAG           Tag name or branch to pull from. Defaults to master.
+
+    --dest=DEST         Destination of tarball. Output to stdout if no destination given.
+
+    --username=USER     Username for private Github repositories.
+
+    --password=PASS     Password for private Github repositories.
```

Comment:
Tomdoc would have these formatted differently.  No spaces at the beginning, a single hyphen separating the option from the description.  Tomdoc.sh uses `$1` instead of named placeholders (the `<repository>` thing).

```
$1              - Path to GitHub repository.
--tag=TAG       - Tag name or branch to pull from.  Defaults to master.
--username=USER - Username for private GitHub repositories.
--password=PASS - Password for private GitHub repositories
```